include/constants/map_types.h

```diff
@@ -0,0 +1,28 @@
+#ifndef GUARD_CONSTANTS_MAP_TYPES_H
+#define GUARD_CONSTANTS_MAP_TYPES_H
+
+#define MAP_TYPE_0           0
+#define MAP_TYPE_TOWN        1
+#define MAP_TYPE_CITY        2
+#define MAP_TYPE_ROUTE       3
+#define MAP_TYPE_UNDERGROUND 4
+#define MAP_TYPE_UNDERWATER  5
+#define MAP_TYPE_OCEAN_ROUTE 6
+#define MAP_TYPE_7           7
+#define MAP_TYPE_INDOOR      8
+#define MAP_TYPE_SECRET_BASE 9
+
+enum
```

Comment:
If these ever need to be included in an asm file, this enum must first be converted into `#define` tokens.

Reply:
my bad, should not have copied this one